src/messages/GE.json

```diff
@@ -316,5 +316,6 @@
   "Toggle selection for all items": "Toggle selection for all items",
   "Toggle selection": "Toggle selection",
   "Row checkbox": "Row checkbox",
-  "Administrator permission": "Admin of the tenant needs to authorize first before you can consent to this permission"
+  "Administrator permission": "Admin of the tenant needs to authorize first before you can consent to this permission",
+  "adaptive cards designer": "adaptive cards designer."
```

Comment:
Should this start with a capital letter? 
`Adaptive card designer`

Reply:
I am not sure as I was using it in the middle of a sentence.
Is there a rule about this?

Reply:
I meant
`"Adaptivate card designed": "adaptive card designer"`

For consistency with other key:value pairs

Reply:
Ah okay! Got it